public/app/features/search/page/SearchPage.tsx

```diff
@@ -0,0 +1,137 @@
+import { DataFrameView, LoadingState, NavModelItem } from '@grafana/data';
+import { Input } from '@grafana/ui';
+import React, { useMemo, useState } from 'react';
+import Page from 'app/core/components/Page/Page';
+import { useAsync } from 'react-use';
+import { buildStatsTable, filterDataFrame, getDashboardData } from './data';
+import { config, PanelRenderer } from '@grafana/runtime';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { DashboardResult } from './types';
+
+export default function SearchPage() {
+  const node: NavModelItem = {
```

Comment:
Doesn't seem like it needs to be inside component.

Reply:
:100: @nmarrs will fix it :)  The goal here is just to prove that this approach can work